### DISCUSSION TITLE
Integration as a seller

### DISCUSSION THREAD

**Discussion by saurabhes** [ONDC-Official/ONDC-FIS-Specifications#69] [General]
Hi 

As a seller do I need to expose all these APIs at my end? is it a kind of deep integration?
How will the decision-making and KYC work? Will it happen on the buyer app or seller have to do this like digilocker integration etc..

Also, saw some form requests in the API doc, how does these form work for us?

**Comment by 92shreyansh** [comment]
@Antriksh-Parmar Please share the community call tracker here.

@saurabhes This tracker has all the recording of our previous community call where these aspects have been covered in detail.
